add-on/chrome/modules/MozLoopPushHandler.jsm

```diff
@@ -764,62 +764,14 @@ var MozLoopPushHandler = {
     // For tests, use the mock instance.
     this._pushSocket = new PushSocket(this._mockWebSocket);
 
-    let performOpen = () => {
-      consoleLog.info("PushHandler: attempt to open websocket to PushServer: ", this.pushServerUri);
-      this._pushSocket.connect(this.pushServerUri,
-                               (aMsg) => this._onMsg(aMsg),
-                               () => this._onStart(),
-                               (aCode, aReason) => this._onClose(aCode, aReason));
-    };
+    this.pushServerUri = Services.prefs.getCharPref("dom.push.serverURL");
```

Comment:
I think we should move this into the `MozLoopPushHandler#initialize` since we only really need to do it once on startup.